custom-recipes/neo4j-export-nodes/recipe.json

```
@@ -148,6 +148,27 @@
             "description": "Nodes with this label will be deleted before run",
             "defaultValue": false,
             "visibilityCondition": "model.expert_mode"
+        },
+        {
+            "label": "Handling NULL Values",
```

Let's use the wording from pandas
```suggestion
            "label": "Handling NaN Values",
```